src/ui/layout/mct-tree.vue

```diff
@@ -710,25 +710,25 @@ export default {
                 }
             });
         },
-        async aggregateSearchResults(results, abortSignal) {
+        aggregateSearchResults(results, abortSignal) {
```

Comment:
Any specific reason to get rid of the async/await here?

Reply:
The function doesn't return a `Promise` and it no longer contains any `await` expressions (they were blocking search results from rendering). `Await`-ing on this function would be meaningless.

Reply:
Hehe... let me rephrase - You replaced the async/await with a .then() approach. I was curious about why?

Reply:
Oh! Right, sorry. `getOriginalPath` triggers a bunch of requests to the server. Using `await` inside this array meant that each call to `getOriginalPath` could not even start until the previous one had resolved. This was blocking search results unnecessarily, and preventing batching.

Reply:
Ok cool thanks! :)